src/FSLibrary/MaxTPSTest.fs

```diff
@@ -137,13 +146,45 @@ let maxTPSTest (context: MissionContext) (baseLoadGen: LoadGen) (setupCfg: LoadG
 
     let tier1 = List.filter (fun (cs: CoreSet) -> cs.options.tier1 = Some true) allNodes
 
+    // On smaller networks, run loadgen on all nodes to better balance the overhead of load generation
+    let loadGenNodes = if List.length allNodes > smallNetworkSize then tier1 else allNodes
+
+    let isLoadGenNode cs = List.exists (fun (cs': CoreSet) -> cs' = cs) loadGenNodes
+
+    let allNodes =
+        match context.numPregeneratedTxs, context.genesisTestAccountCount with
+        | Some txs, Some accounts ->
+            let loadGenCount = List.length loadGenNodes
```

Comment:
A couple comments here might be helpful. Is my understanding correct?

1. Divy up account offsets to each node generating load based on how many genesis accounts we have
2. Assign account offsets and the number of TXs to preload per load generating node?

Reply:
Added.